### DISCUSSION TITLE
How do I connect PlanetScale to Django ?

### DISCUSSION THREAD

**Discussion by pintu0789** [planetscale/discussion#21] [Documentation] [answered]
Does planetscale db has supports for django app ? Can someone point me to a doc or some sort which can show me an example of onboarding to test for a django based app use case

**Comment by Anders-PlanetScale** [comment, marked as answer]
Howdy,

We don't currently have any specific Django documentation but you can follow our documentation [here](https://docs.planetscale.com/tutorial/connect-any-application) which has generic connection details and steps outlined to connect your application to psdb.

**Comment by Alastorpray** [comment]
Django with planetscale works very well, I was researching a few days and it was tedious since I am new to django but I found a way to make it work correctly, I have even uploaded the project to a server.

I will go up some steps to be able to help the community.
